### DISCUSSION TITLE
Allow .svg upload for images in media library and image user template blocks

### DISCUSSION THREAD

**Discussion by sverraest** [forkcms/forkcms#3337] [Ideas / Feature Requests]
## Type
- Enhancement

## Problem description

Cannot upload SVG files for image user templates or using media library

## Steps to reproduce

Go to media library or add image to user template, attempt to upload .svg

## Expected behavior / Proposed solutions

Allow configuration of file upload extensions / mimetype

**Comment by Sellato** [comment]
I think it's impossible :( Fork CMS is using Fine Uploader, but Fine Uploader is dead.

> Fine Uploader is shutting down

So, no SVG, no WEBP and so on.
It's will be great if I'm wrong :)

**Comment by carakas** [comment]
feel free to create a PR that could solve this

**Comment by sverraest** [comment]
I've solved this in a hack-ish way, will see if I can improve.
Biggest issue is that there's no point in resizing a .svg which seems to be linked to the image upload.
